### ISSUE TITLE
Broaden deCONZ device detection (on Docker/Synology)

### ISSUE THREAD

**Issue by systemcrash** [zigpy/zigpy-deconz#150]
https://github.com/zigpy/zigpy-deconz/blob/9ee4440f267e10066541c23ff956a22fb2523d2f/zigpy_deconz/zigbee/application.py#L384

Not entirely sure  how this detection works, but I run a docker image on my Synology. It always presents the USB device as:

`/dev/ttyACM0`

If the referenced Regex is what does the detection magic, I propose that it be extended to e.g.:

```python
is_gpio_device = re.match(r"/dev/tty(S|AMA|ACM)\d+", device_path)
```

Or is this not what this regex is for?

**Comment by Adminiuga** [comment]
fixed by db2861d0dc9a0ebf3b8331490c1652d66b26e156

**Comment by systemcrash** [comment]
OK thank you 😄
